packages/app-accounts/src/Editor.tsx

```diff
@@ -314,9 +325,26 @@ class Editor extends React.PureComponent<Props, State> {
     this.setState(
       this.createState(null),
       () => {
-        keyring.forgetAccount(
-          current.address()
-        );
+
+        try {
+          keyring.forgetAccount(
+            current.address()
+          );
+
+          onStatusChange({
+            action: 'forget',
+            success: true,
+            value: current.address(),
+            message: 'Forgot'
+          } as ActionStatus);
+        } catch (e) {
+          onStatusChange({
```

Comment:
This duplicates what happens on line 334, only the status is different? Rather set the status inside the try/catch and set the status externally to it.